template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/utils.py.tmpl

```diff
@@ -1,22 +1,17 @@
-"""
-This module contains utils shared between different notebooks
-"""
-import json
-import mlflow
-import os
```

Comment:
Thanks for removing these imports. I wonder why we had them if we didn't need them